example.txt

```diff
@@ -0,0 +1,3 @@
+-->Git branch = shoing branches
```

Comment:
you don't need that assignment = and also remove the declaration of that git command